src/components/Match/MatchHeader/MatchHeader.css

```diff
@@ -0,0 +1,68 @@
+@import "../../palette.css";
+
+.gme {
+  text-align: center;
+
+  & span {
+    text-transform: uppercase;
+    display: block;
+  }
+
+  & .gameMode {
+    font-size: var(--fontSizeMedium);
+  }
+  & .duration {
+    font-size: 28px;
+  }
+  & .ended {
+    font-size: var(--fontSizeSmall);
+    color: var(--colorMutedLight);
+    margin-top: 3px;
+    & > div {
+      display: inline-block;
+    }
+  }
+}
+
+.winner {
+  font-size: 28px;
+  padding-top: 10px;
+  filter: drop-shadow(0 0 15px var(--colorYelor));
+}
+.score {
+  font-size: 48px;
+}
+
+.team {
+  & svg {
+    width: 32px;
+    height: 32px;
+    margin: 0 15px;
+    vertical-align: sub;
+  }
+}
+
+.radiant {
+  composes: team;
```

Comment:
Can we make this responsive? Right now, it sort of breaks after 760px, and it starts to look sort of bad at ~820ish